Client/Application/AppDelegate.swift

```diff
@@ -203,8 +203,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         sendCorePing()
 		
-        //Init the bloomFilter
-        _ = BloomFilterManager.sharedInstance
+        //Turn on the bloomFilter
+        BloomFilterManager.sharedInstance.turnOn()
```

Comment:
I think it is a heavy operation to do on Main thread while the app is starting, could you run this in a background thread?